DrawerKit/DrawerKit.xcodeproj/project.pbxproj

```diff
@@ -343,7 +343,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.3;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.2;
```

Comment:
Is this change intentional?

Reply:
Yes. It makes it easier to test `DrawerKit` in the simulator.